homeassistant/components/sensibo/quality_scale.yaml

```diff
@@ -0,0 +1,77 @@
+rules:
```

Comment:
What's
```py
    SensiboMotionBinarySensorEntityDescription(
        key="is_main_sensor",
        translation_key="is_main_sensor",
        entity_category=EntityCategory.DIAGNOSTIC,
        value_fn=lambda data: data.is_main_sensor,
    ),
```

Reply:
If the motion sensor is the main sensor for temp/humidity or if it's the climate device itself.